packages/vuetify/src/components/VTreeview/VTreeview.ts

```diff
@@ -68,6 +68,21 @@ export default mixins(
       type: Array,
       default: () => ([])
     } as PropValidator<NodeArray>,
+    search: String,
+    customFilter: {
+      type: Function,
+      default: (items: any[], search: string, idKey: string, textKey: string, childrenKey: string) => {
```

Comment:
`customFilter` in `data-iterable` returns the list of filtered items, here it returns excluded. I think it should be done here in a similar way for consistency (and also for consistencty `filter` prop would be nice)

Reply:
This was done for a reason. It has to do with the fact that the treeview creates an internal state of all of the nodes including the selected, active states. It's possible that custom-filter could be made to return filtered items but that would require a much larger rewrite.

I also didn't include filter prop because we've had talks previously of changing the way filter and custom-filter works in data-iterable because they can be pretty confusing.